cmd/ttn-lw-stack/commands/start.go

```diff
@@ -170,11 +170,28 @@ var startCommand = &cobra.Command{
 
 		redisConsumerID := redis.Key(host, strconv.Itoa(os.Getpid()))
 
+		transport, err := c.HTTPTransport(ctx)
+		if err != nil {
+			logger.WithError(err).Error("Failed to setup HTTP transport")
```

Comment:
Should we continue execution?

Reply:
This should indeed halt the execution, please `return err`.